packages/turf-rbush/README.md

```diff
@@ -0,0 +1,217 @@
+# GeoJSON RBush used in TurfJS
+
+This fork of geojson-rbush is configured and published for the sole purpose of importing it to [TurfJS](https://github.com/Turfjs/turf).
+
+More information about the original library: https://github.com/DenisCarriere/geojson-rbush
+
+> This library is not intended to be used outside of TurfJS
```

Comment:
I'm not sure we need to persist this warning? Its kind of a nice optimization on top of the rbush if you're dealing with geojson.